web/app/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require("express");
+var router = express.Router();
+
+// frontend routes =========================================================
+// route to handle all angular requests
+router.get("*", function(req, res) {
+  res.sendfile("./public/index.html");
+});
+
+module.exports = router;
```

Comment:
So, this confused for a while...

I saw this and was like "You aren't requiring this file from anywhere and adding it to the app.use. So this shouldn't work"
Then I tested it, and it worked.... which confused me ever so.

Turns out, this line saved you _partially_
```
// server.js line 27
app.use(express.static(__dirname + "/public"));
```
If you go to the homepage/root, it will default give you the index.html file, and since this is Angular, it will also give you every other view you need , such as the stats page, to make the Single Page App. Meaning you can press the stats page button and it will still work, but if you go straight to the /stats page, you would get a 404.
This confused for quite a bit

tl;dr fix
require and app.use this file in server.js